src/video.js

```diff
@@ -35,15 +35,16 @@ export const hasNonVideoBidder = adUnit =>
 export function isValidVideoBid(bid, {index = auctionManager.index} = {}) {
   const videoMediaType = deepAccess(index.getMediaTypes(bid), 'video');
   const context = videoMediaType && deepAccess(videoMediaType, 'context');
+  const useCacheKey = videoMediaType && deepAccess(videoMediaType, 'useCacheKey');
   const adUnit = index.getAdUnit(bid);
 
   // if context not defined assume default 'instream' for video bids
   // instream bids require a vast url or vast xml content
-  return checkVideoBidSetup(bid, adUnit, videoMediaType, context);
+  return checkVideoBidSetup(bid, adUnit, videoMediaType, context, useCacheKey);
 }
 
-export const checkVideoBidSetup = hook('sync', function(bid, adUnit, videoMediaType, context) {
-  if (videoMediaType && context !== OUTSTREAM) {
+export const checkVideoBidSetup = hook('sync', function(bid, adUnit, videoMediaType, context, useCacheKey) {
```

Comment:
I don't think you need the additional parameter - you already have `videoMediaType`.

Reply:
but videoMediaType is true for in- and outstream. I just want to add a switch to outstream, so I have to pass the value of that switch.

We have instream where cacheKey always needs to be true and outstream where now it always is false, both with videoMediaType set to true, so passing useCacheKey only respected for outstream seems to me the only way here.

Reply:
I mean you can move line 38 here (`useCacheKey = videoMediaType && deepAccess(videoMediaType, 'useCacheKey')`), since `videoMediaType` is the same object.